client/me/purchases/vat-info/index.tsx

```diff
@@ -54,13 +53,17 @@ export default function VatInfoPage() {
 	/* This is the title of the support page from https://wordpress.com/support/vat-gst-other-taxes/ */
 	const taxSupportPageLinkTitle = translate( 'VAT, GST, and other taxes' );
 
-	const handleOpenCenterChat = useCallback( async () => {
-		clearHelpCenterZendeskConversationStarted();
-		setNavigateToRoute( '/odie' );
-		setShowHelpCenter( true );
-		await resetSupportInteraction();
-		reduxDispatch( recordTracksEvent( 'calypso_vat_details_support_click' ) );
-	}, [ reduxDispatch, resetSupportInteraction, setNavigateToRoute, setShowHelpCenter ] );
+	const handleOpenCenterChat = useCallback(
+		async ( e: React.MouseEvent< HTMLAnchorElement > ) => {
```

Comment:
I think we can be even more generic with the event type, there is no need to be that verbose:

```suggestion
		async ( e: MouseEvent ) => {
```

Reply:
` client/me/purchases/vat-info/index.tsx(143,12): error TS2322: Type '(e: MouseEvent) => Promise<void>' is not assignable to type 'MouseEventHandler<HTMLAnchorElement>'` I see this now 😄

Reply:
I reverted this change since I saw this type check error.